### ISSUE TITLE
Node v12 Support

### ISSUE THREAD

**Issue by nodehack** [JacobFischer/netlinkwrapper#8]
Looks like V8 changes are causing build issues when using node version 12.
```
> netlinkwrapper@1.1.1 install /Projects/thingy/node_modules/netlinkwrapper
> node-gyp rebuild

  CXX(target) Release/obj.target/netlinksocket/netlinksocket.o
In file included from ../netlinksocket.cc:2:
../netlinkwrapper.h:11:30: error: no template named 'Handle' in namespace 'v8'
        static void Init(v8::Handle<v8::Object> exports);
                         ~~~~^
../netlinksocket.cc:6:6: error: variable has incomplete type 'void'
void InitAll(Handle<Object> exports)
     ^
../netlinksocket.cc:6:21: error: 'Object' does not refer to a value
void InitAll(Handle<Object> exports)
                    ^
/Users/nodehack/.node-gyp/12.3.1/include/node/v8.h:3356:17: note: declared here
class V8_EXPORT Object : public Value {
                ^
../netlinksocket.cc:6:14: error: use of undeclared identifier 'Handle'
void InitAll(Handle<Object> exports)
             ^
../netlinksocket.cc:6:29: error: use of undeclared identifier 'exports'
void InitAll(Handle<Object> exports)
                            ^
../netlinksocket.cc:6:37: error: expected ';' after top level declarator
void InitAll(Handle<Object> exports)
                                    ^
                                    ;
6 errors generated.
```

**Comment by JacobFischer** [comment]
Hmmm, I guess I'll have to investigate what changed between v10 and v12 such that #4 is not enough to fix it. Thanks for the heads up!

**Comment by JacobFischer** [comment]
Fixed via #9
